frontend-student/src/components/coursePage/QuizCompletionModal.jsx

```
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function QuizCompletionModal({ isOpen, onClose }) {
+  if (!isOpen) return null;
+
+  return (
+    <div className='fixed inset-0 flex items-center justify-center z-50'>
+      <div
+        className='fixed inset-0 bg-black bg-opacity-50'
+        onClick={onClose}
+      ></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)